PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/PBMPrebidParameterBuilder.m

```diff
@@ -86,6 +86,16 @@ - (void)buildBidRequest:(nonnull PBMORTBBidRequest *)bidRequest {
     if (adFormats.count >= 2) {
         bidRequest.extPrebid.targeting[@"includeformat"] = [[NSNumber alloc] initWithBool:YES];
     }
+
+    if(Prebid.shared.includeWinners)
```

Comment:
It would be nice to add unit tests for these new properties to `PrebidParameterBuilderTest` to avoid regression in the future.

Reply:
@AbrahamArmasCordero, please, add unit tests so we can finalize changes and merge the PR.

Reply:
done